src/commands/quoteCommand.ts

```
@@ -32,27 +32,30 @@ export class QuoteCommand implements Command {
         const messageReference = await this.getMessageReference(parsedUserCommand);
         if (messageReference == undefined) return;
 
-        const message = await this.getMessageByReference(
-            parsedUserCommand.client,
-            messageReference
-        );
+        const client = parsedUserCommand.client;
+        const guild = await client.guilds.fetch(messageReference.guildID);
+        if (guild == undefined) return;
```

Unfortunately this won't work, guilds.fetch either returns the guild or throws, so you're gonna have to wrap it in a try catch instead of a null check.

---

Ah, good catch! Thanks, will fix!

---

alright, fixed.